packages/flutter/lib/src/widgets/inherited_theme.dart

```diff
@@ -87,10 +99,10 @@ abstract class InheritedTheme extends InheritedWidget {
   static CapturedThemes capture({required BuildContext from, required BuildContext? to}) {
     if (from == to) {
       // Nothing to capture.
-      return CapturedThemes._(const <InheritedTheme>[]);
+      return CapturedThemes._(const <InheritedTheme<dynamic, dynamic>>[]);
```

Comment:
All of the dynamics here are a red flag at first glance at least. https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-using-var-and-dynamic

Reply:
Okay, I will migrate to may be `Object?`.